<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StereoVisionApp::FixedStereoSequenceEditor</class>
 <widget class="QWidget" name="StereoVisionApp::FixedStereoSequenceEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="folderLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="selectFolderButton">
        <property name="text">
         <string>open folder...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="right_suffix_edit">
        <property name="text">
         <string>_right</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="left_suffix_edit">
        <property name="text">
         <string>_left</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="rgb_suffix_edit">
        <property name="text">
         <string>_rgb</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="left_suffix_label">
        <property name="text">
         <string>Left suffix:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="right_suffix_label">
        <property name="text">
         <string>Right suffix:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="rgb_suffix_label">
        <property name="text">
         <string>RGB suffix:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="imgListView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
